src/presentation/pages/NoteSettings.vue

```diff
@@ -17,6 +17,17 @@
       type="primary"
       @click="regenerateHash"
     />
+    <form-field
+      v-model="newParentURL"
```

Comment:
so it will be empty by default? Maybe we need to display current parent in there?